fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCSchemaResolver.java

```diff
@@ -89,8 +89,18 @@ public List<Column> convertToSRTable(ResultSet columnSet) throws SQLException {
                     columnSet.getString("TYPE_NAME"),
                     columnSet.getInt("COLUMN_SIZE"),
                     columnSet.getInt("DECIMAL_DIGITS"));
+
+            String comment = "";
+            // Add try-cache to prevent exceptions when the metadata of some databases does not contain REMARKS
```

Comment:
typo: try-catch.  and please fix the code style.

Reply:
done